include/Piano.h

```diff
@@ -65,6 +65,10 @@ class Piano final : public Model
 	static bool isWhiteKey(int key);
 	static bool isBlackKey(int key);
 
+	static const unsigned int WhiteKeysPerOctave = 7;
+	static const unsigned int BlackKeysPerOctave = 5;
+	static const unsigned int NumWhiteKeys = 75;
+	static const unsigned int NumBlackKeys = 53;
```

Comment:
Is there a way to gain this through math and our constants?

Reply:
I don't think they could be derived "mathematically"; the first two are arbitrary constants that describe how many notes from one 12-EDO octave are deemed "special" when making a piano keyboard, the other two count how many of these special keys are available in total. And since the black-white order is also arbitrary, there is no easy mathematical formula that says how many special keys are in the last, incomplete octave.

It could be determined by an algorithm though, by simply checking `Piano::KeyTypes KEY_ORDER[]` (defined in `Piano.cpp` for some reason, not the header file), i.e. running `isBlackKey()` and `isWhiteKey()` for all keys and, well, counting them. Or, more efficiently, count all complete octaves using a simple formula, and finishing the count by looping through the last, incomplete octave.

Either way seems pretty convoluted and probably would have to run in the constructor (can you initialize a constant in the constructor? Actually, can you initialize static _anything_ in a constructor, since there is only one "instance"?)

Reply:
All good then. Just wanted to check. 👍